### PR TITLE
feat: [CAR-4764] Made overrides templates

### PR DESCRIPTION
**Pull request by TravnikovDev**
**PR description:**

Added CLI tools which generate boilerplate code from templates (components, tests, hooks, styles, layouts, etc.), replacing manual copy-paste and editing with a simple terminal command. Used plop.js library

**Checklist:**

- [x] I ran this code locally
- [x] I wrote the necessary tests
- [x] My code follows the [style guidelines](http://bit.ly/sd-web-style-guide)
- [x] I followed the [instructions](http://bit.ly/sd-web-pr) to create a pull request

**JIRA card:**

[[CAR-4764] Create CLI tools for fast component creation](https://superdispatch.atlassian.net/browse/MSG-23)

**Should know about:**

Added description into README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -162,6 +163,7 @@
     "mockdate": "3.0.5",
     "npm-run-all": "4.1.5",
     "playroom": "0.27.9",
+    "plop": "^3.1.2",
```
[the remark this answers is not in the document]
Comment (TravnikovDev) [reply package.json]: Updated

File: plop-templates/README.hbs
[the remark this answers is not in the document]
Comment (TravnikovDev) [reply plop-templates/README.hbs:1]: Moved

**Review by timurhamitov — APPROVED** [review]

File: scripts/plop-templates/customHook.hbs
Comment (doniyor2109) [inline scripts/plop-templates/customHook.hbs:1]: Do we use it somewhere?

File: scripts/plop-templates/lab-component/component.hbs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    // Implement your component's functionality here
+
+    return (
+      <Styled{{pascalCase name}} ref={ref} data-testid="{{camelCase name}}" {...props}>
+        {/* Render your component's children or other UI elements here */}
+      </Styled{{pascalCase name}}>
+    );
+  },
+);
```
Comment (doniyor2109) [inline scripts/plop-templates/lab-component/component.hbs:28]: Can we add displayName as well?

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

File: scripts/plop-templates/lab-component/component.hbs (hunk 2 above)
Comment (TravnikovDev) [reply scripts/plop-templates/lab-component/component.hbs]: Added

File: scripts/plop-templates/customHook.hbs
Comment (TravnikovDev) [reply scripts/plop-templates/customHook.hbs:1]: It's just template. I've left it as a showcase. Maybe someone will extend it in future

Comment (doniyor2109) [reply scripts/plop-templates/customHook.hbs:1]: @TravnikovDev Lets remove it for now.

**Review by doniyor2109 — APPROVED** [review]

**Pull request merged**